README.md

```diff
@@ -1 +1,198 @@
-# Resort_Reserve_Back-end
\ No newline at end of file
+<a name="readme-top"></a>
+<h1 align='center'>Final Capstone - Resort Booking</h1>
+
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [ Rails Blog ](#-My-Blog-App-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Database](#database)
+    - [Usage](#usage)
+    - [Testing](#testing)
+    - [Deployment](#deployment)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+
+# Rails Blog <a name="about-project"></a>
+> Resort booking is a wepbage made with rails and react, for users to book different resorts.
+
+## 🛠 Built With <a name="built-with"></a>
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Ruby On Rails with React</summary>
+  <ul>
+    <li><a href="">Postgresql</a></li>
+  </ul>
+</details>
+
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Visit our webpage here.
+
+- [Live Demo Link](https://resort-booking-back-end.onrender.com)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Key Features <a name="key-features"></a>
+
+- Create/remove a booking for a resort
+- See a list of all possible resorts
+- Admins are able to add resorts to the list
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- Mac or PC
+- Understanding OOP principles
+- Have the following installed on your machine
+    - Ruby
+    - Rails
+    - PostgreSQL
+    - React
+
+### Setup
```

Comment:
- [ ] Your project requires multiple custom procedures in order to be run correctly.

1. The project depends on a _.env_ file that must be created and setup locally for things like the jwt_secret and the like:  

![image](https://github.com/Thinus01/Resort_Booking_Back-end/assets/57421823/2a5d3217-6550-42c4-bebe-b4f29c1fc1ef)

Kindly include specific instructions on your readme about how to do this with the correct data.

2. Is it needed to run `rails secret`?  Does the master.key need to be created?  Add instructions for any of this that is true.


Kindly, include on your readme any custom/special procedures that your users need to do in order to run your project and your tests correctly.

That way, all your users (including potential employers) will be able to run your project smoothly. 💪 

_note:  This means I was not able to create the database and run the tests.  This will be taken a look at in the next review.  Also, please make sure all your tests pass if run according to your instructions._